site/docs/docs.md

```diff
@@ -5,9 +5,14 @@ title: Overview
 permalink: /docs/
 ---
 
-This documentation describes [Vega-Lite specifications](#spec) and the functionalities of the [Vega-Lite library](#lib).
+**Vega-Lite** is a high-level visualization grammar.  It provides a concise JSON syntax for supporting rapid generation of visualizations to support analysis.  Vega-Lite can serve as a declarative format for describing and creating data visualizations.  Vega-Lite specifications can be compiled to a lower-level, more detailed specifications in the form of [Vega](http://vega.github.io/vega) specifications and rendered using Vega's compiler.
```

Comment:
I don't think it's good that the docs repeat what we already write on the homepage. Do we want to keep the home page shorter then?